Makefile

```diff
@@ -55,10 +55,10 @@ protobuf:
 # Build coin definitions
 coins:
 	# make submodules
-	./submodules/trezor-common/tools/cointool.py dump -p -d connect -e icon -e cooldown -e github -e key -e maintainer -e uri_prefix -e version_group_id -e website -e links -e duplicate -e wallet -e bitcore -e confidential_assets -e negative_fee -o ./src/data/coins.json
+	../trezor-firmware/common/tools/cointool.py dump -p -d connect -e icon -e cooldown -e github -e key -e maintainer -e uri_prefix -e version_group_id -e website -e links -e duplicate -e wallet -e bitcore -e confidential_assets -e negative_fee -o ./src/data/coins.json
```

Comment:
do not change the path to submodule, connect is strictly bound with commit in "trezor-common" submodule
To regenerate coins this submodule needs to be updated first